Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
+    turbo-rails (1.1.1)
```

Comment:
### [turbo-rails](https://github.com/hotwired/turbo-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.1 → 1.1.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/turbo-rails/1.0.1/1.1.1)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/hotwired/turbo-rails/compare/v1.0.1...v1.1.1).

These are the first 10 commits:
- (2a2a5ec) [Use where on Windows instead of which (#299) (#300)](https://github.com/hotwired/turbo-rails/commit/2a2a5ec12e4233cf33e5d3f0a6855834f4e5e4ec)
- (a038371) [Test on Rails 7.0](https://github.com/hotwired/turbo-rails/commit/a0383713d0428169d9d62b829745da8f81442a18)
- (66ca1d7) [fix tests for rails 7.0 (#320)](https://github.com/hotwired/turbo-rails/commit/66ca1d7758ab271f3747a79a3a9be8a159357782)
- (4283962) [add targets to assert_{,no_}turbo_stream (#321)](https://github.com/hotwired/turbo-rails/commit/4283962089c998fbe5c77bf0db374efa52cb6a46)
- (862050c) [Fix CI with Rails main (#314)](https://github.com/hotwired/turbo-rails/commit/862050cbea007eb44f9c64ee511df6878858d449)
- (395fef0) [Problem: inability to pass additional parameters to stream c…](https://github.com/hotwired/turbo-rails/commit/395fef0e883a6723dc8c7cc6f46745f35026f533)
- (0178d18) [Fix breaking in non-Zeitwerk environment (#304)](https://github.com/hotwired/turbo-rails/commit/0178d184f1bf430ca83b93496377fc807a2d3d5e)
- (d4e26fe) [Add possibility to nest turbo frames (#296)](https://github.com/hotwired/turbo-rails/commit/d4e26fec0ed02e06cd3ab26882dc234fccd2972a)
- (b9b87fc) [Document the turbo_frame_tag helper with multiple arguments …](https://github.com/hotwired/turbo-rails/commit/b9b87fc627d9148899970deea01069875f15db3f)
- (5d6fe3d) [Fix missing `activejob` dependency (#331)](https://github.com/hotwired/turbo-rails/commit/5d6fe3de5373cc49d2138d4133195a22e834d57e)


</details>